File: lab6/src/main/java/Helmet.java
public class Helmet extends Equipment{

    private int size;

    public Helmet(String ProduceName, double weight, SportType sportType, int size) {
        super(ProduceName, weight, sportType);
        this.size = size;
    }

    public int getSize() {
        return size;
    }

    public void setSize(int size) {
        this.size = size;
    }

    @Override
    public String toString() {
        return "Helmet {" +
                "size = " + size + " " +
                super.toString() +
                "}\n";
    }
}

File: lab 13-14/model/helmet.py
from equipment import Equipment


class Helmet(Equipment):

    def __init__(self, produce_name, weight,  size, type):
        super(Helmet, self).__init__(produce_name, weight, type )
        self.size = size

    def __str__(self):
        return super(Helmet, self).__str__() + ', size = ' + str(self.size)
